stable/artifactory/templates/artifactory-statefulset.yaml

```diff
@@ -276,6 +276,11 @@ spec:
         - containerPort: {{ .Values.artifactory.javaOpts.jmx.port }}
         {{- end }}
         volumeMounts:
+      {{- if .Values.customCA.enabled }}
```

Comment:
Is using the features for:
customVolumes and customVolumesMounts not good enough for you? You can achieve the same result with the more generic feature which already exists in the Artifactory chart. Not sure it makes sense to add this specifically.

Reply:
First: I am all for using existing functionality. It is more then good enough for me :) Before I put the time in for this PR, I contacted jFrog support, and they didn't mention the functionality and I couldn't find it in the documentation. 

Custom mounts in this case likely won't work based on the documentation available. Specifically because you need to add a single file to the folder, and the custom mount likely will over write all existing files. 

Adding a custom CA cert (single, not multiple) is required functionality for any environment that runs not connected to the internet.

My customers pay a ton of money for Artifactory, and this feature, whether added via this PR or another, is required for their continued use of Artifactory.

Reply:
@evannorstrand First of all. thank you for your time and your contribution. 
I apologise that the support team didn't tell you about this option, but it does exist and here is a link to the documentation: 
customVolumes: https://github.com/jfrog/charts/blob/master/stable/artifactory/README.md#custom-volumes

When you use `customVolumes` along with with `customVolumeMounts`, you can achieve your desired behaviour. 
Regarding your concern with mounting a single file - you can use the `subPath` option with the `customVolumeMounts` when mounting a configMap, same as you did.
Here's an example of a values file you can use:

```
artifactory:
  ## Set up your volume to the CA config map
  customVolumes: |
    - name: ca-configmap
      configMap:
        name: my-ca-configmap
  ## Mount the volume to the certs directory using the subPath method
  customVolumeMounts: |
    - name: ca-configmap
      mountPath: /etc/ssl/certs/custom-ca.crt
      subPath: custom-ca.crt
```

I agree that mounting a custom CA is an important feature, but if we already have a generic solution in place for that, I think it makes sense to use it.

You are correct though that is feature is missing from Xray. I think that this should definitely be added, if you're interested in suggesting a PR. If not, I'll be surely work on that next week :)

Please let me know if that solves your issue with Artifactory, if so, I will add that feature to Xray as well.

Reply:
For now I will close this PR, make the suggested changes in both this PR and the other PR that I submitted, and push a new PR. 

We need the functionality sooner rather then later for our customers to move forward, and i don't mind contributing back :)